prow/crier/reporters/slack/reporter.go

```diff
@@ -86,32 +95,38 @@ func (sr *slackReporter) GetName() string {
 
 func (sr *slackReporter) ShouldReport(pj *v1.ProwJob) bool {
 	logger := sr.logger.WithFields(pjutil.ProwJobFields(pj))
-	// if a user specifically put a channel on their job, they want
-	// it to be reported regardless of what other settings exist
-	if _, set := jobChannel(pj); set {
-		logger.Debugf("reporting as channel is explicitly set")
-		return true
+	jobCfg := jobConfig(pj)
+	prowCfg := sr.config(pj.Spec.Refs)
+	// If a user specifically put a channel on their job, they want
```

Comment:
Is this comment still true?

Reply:
I updated the comment: `regardless of what other settings exist` => `regardless of the job types setting`